schematic/manifest/generator.py

```diff
@@ -1236,18 +1236,19 @@ def _gather_all_fields(self, fields, json_schema):
         )
         return required_metadata_fields
 
-    def get_empty_manifest(self, json_schema_filepath=None, strict=None):
+    def get_empty_manifest(self, strict: Optional[bool], json_schema_filepath: str=None):
         """Create an empty manifest using specifications from the
         json schema.
         Args:
+            strict (bool): strictness with which to apply validation rules to google sheets.
```

Comment:
Can you just add here: True, blocks incorrect entries, False, raises a warning

Reply:
Yes. Thank Mialy! I have addressed that.